source/draw/RasterUv.ooc

```diff
@@ -61,13 +61,13 @@ RasterUv: class extends RasterPacked {
 
 	distance: func (other: Image) -> Float {
 		result := 0.0f
-		if (!other)
+		if (!other || (this size != other size))
 			result = Float maximumValue
-//		else if (!other instanceOf?(This))
-//			FIXME
-//		else if (this size != other size)
-//			FIXME
-		else {
+		else if (!other instanceOf?(This)) {
+			converted := This convertFrom(other as RasterImage)
+			result = this distance(converted)
+			converted referenceCount decrease()
+		} else {
```

Comment:
I don't know. I have the same problem with the bilinear rescaling function I pushed recently, it will look almost the same in every class, but I don't know how to generalize it yet.